drivers/bt_ll_nrfxlib/clock_control/Kconfig

```diff
@@ -0,0 +1,77 @@
+#
+# Copyright (c) 2018 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+config CLOCK_CONTROL_NRF_LL_NRFXLIB
+	bool "Nordic BLE controller clock control driver"
+	default y
+	select CLOCK_CONTROL_NRF_FORCE_ALT
+
+choice CLOCK_CONTROL_NRF_SOURCE
+	prompt "32KHz clock source"
```

Comment:
32 KHz (space missing)
occurs in other places as well

Reply:
Fixed.